### ISSUE TITLE
which feature_perturbation method in TreeSHAP uses marginal contributions rather than conditional? (Urgent! - viva soon)

### ISSUE THREAD

**Issue by Ariel-Flint-Ashery** [shap/shap#2458]
Hi,

In the Interpretable-ML book 9.6.4 (https://christophm.github.io/interpretable-ml-book/shap.html) the author states that "We can use the fast TreeSHAP estimation method instead of the slower KernelSHAP method, since a random forest is an ensemble of trees. But instead of relying on the conditional distribution, this example uses the marginal distribution". 

Which feature_perturbation method of TreeSHAP does this refer to? Or is this a completely different parameter. 

I assumed that it was 'interventional' as this method breaks down correlations, but I'm not certain. For my dataset, using 'tree-path-dependent' is much faster and also shows large importance for two features which are highly correlated (diJ_mass and diJ_deta) but don't actually improve the model by much when used together rather than individually (i.e. I would expect one feature to be high and the other to be low, or both middling, in an importance plot which considered correlation).

![image](https://user-images.githubusercontent.com/91542935/159928601-4a1e77ab-547b-45ee-b7dc-0b0f4a935551.png)

Tl;dr: which feature_perturbation uses marginal contributions rather than conditional expectations?

Thanks!
